doc/tutorial/ui.md

```diff
@@ -0,0 +1,422 @@
+   :width: 100%
+   :alt: Create an Ubuntu 24.04 LTS desktop VM
+   ```
+
+   To run smoothly, the desktop VM needs more RAM.
+   Therefore, navigate to {guilabel}`Advanced` > {guilabel}`Resource limits` and set the {guilabel}`Memory limit` to 4 GiB before you start the instance.
+
+1. Check the list of instances that you created:
+
+   ```{figure} /images/tutorial/instances.png
+   :width: 100%
+   :alt: List of instances
+   ```
+
+   You will see that all but the `third` container are running.
+   This is because you created the `third` container but didn't start it.
+
+   You can start the `third` container by clicking the {guilabel}`Start` button (▷) next to it.
+
+See {ref}`instances-create` for more information.
+
+## Inspect instances
+
+In the list of instances, click on one of the lines to see more information about the respective instance:
+
+```{figure} /images/tutorial/instance_summary.png
+:width: 100%
+:alt: Information about an instance in the instance summary
+```
+
+Click on an instance name to go to the instance detail page, where you can inspect your instance:
+
+- The {guilabel}`Overview` tab shows general information and usage statistics about the instance.
+- The {guilabel}`Configuration` tab contains the instance configuration.
+  For now, just click through to inspect the configuration.
+  We'll do some updates later.
+
+  If you want to see the full instance configuration, go to the YAML configuration:
+
+   ```{figure} /images/tutorial/yaml_configuration.png
+   :width: 100%
+   :alt: YAML configuration of an instance
+   ```
+
+- The {guilabel}`Snapshots` tab shows available snapshots.
+  You can ignore this for now; we'll look into snapshots later.
+- The {guilabel}`Terminal` tab allows you to interact with your instance.
+  For example, enter the following command to display information about the operating system:
+
+      cat /etc/*release
+
+  Or have some fun:
+
+      apt update
+      apt install fortune
+      /usr/games/fortune
+
+  See {ref}`run-commands-shell` for more information.
+
+  ```{note}
+  When you navigate away from the {guilabel}`Terminal` tab, you are asked to confirm.
+  The reason for this confirmation prompt is that the terminal session is not saved, so once you navigate away, your command history is lost.
+  ```
+
+- The {guilabel}`Console` tab is mainly relevant during startup of an instance, and for VMs.
+
+  Go to the instance detail page of the `ubuntu-desktop` VM and check the graphic console:
+
+   ```{figure} /images/tutorial/desktop_console.png
+   :width: 100%
+   :alt: Graphic console of the Ubuntu desktop VM
+   ```
+
+  See {ref}`instances-console` for more information.
+- The {guilabel}`Logs` tab contains log files for inspection and download.
+  Running instances have only limited information.
+  More log files are added if an instance ends up in error state.
+
+  See {ref}`instances-troubleshoot` for more information.
+
+## Stop and delete instances
```

Comment:
related to the previous comment about potentially removing the `second` container creation. This section may need some adjustments